build/lender/Lender.sol

```diff
@@ -50,11 +51,20 @@ contract Lender {
   /// @param m: the address of the marketplace contract
   /// @return bool true if the address was set, false otherwise
   function setMarketPlaceAddress(address m) authorized(admin) external returns (bool) {
-    require(marketPlace == address(0));
-    marketPlace = m;
+    require(address(marketPlace) == address(0));
```

Comment:
do we have a ticket to revisit the require messages? I know we are gunna look at custom errors anyway, so maybe that doesn't matter... :shrug:

Reply:
I just made a ticket for this. 
https://github.com/Swivel-Finance/gost/issues/310